pkg/cloud/scope/session.go

```diff
@@ -89,6 +92,7 @@ func sessionForRegion(region string, endpoint []ServiceEndpoint) (*session.Sessi
 	ns, err := session.NewSession(&aws.Config{
 		Region:           aws.String(region),
 		EndpointResolver: endpoints.ResolverFunc(resolver),
+		UseFIPSEndpoint:  getFIPSStateForRegion(region),
```

Comment:
do we still need to configure `EndpointResolver`? (if yes, is that needed to dKC as well?)